packages/drivers/routerlicious-driver/src/policies.ts

```diff
@@ -3,6 +3,24 @@
  * Licensed under the MIT License.
  */
 
+export enum RouterliciousDriverPerformanceEventName {
+    getVersions = "getVersions",
+    readBlob = "readBlob",
+    // Shredded summaries
+    getSnapshotTree = "getSnapshotTree",
+    // Whole summaries
+    getWholeFlatSummaryTree = "getWholeFlatSummaryTree",
+}
+/**
+ * Number of [eventName] performance events to aggregate into 1 event.
+ * Setting as undefined will disable aggregation for [eventName] events.
+ * Example: { readBlob: 100 } will aggregate every 100 readBlob events into 1 event and log it.
+ */
+type ITelemetryAggregationPolicy = Partial<Record<RouterliciousDriverPerformanceEventName, number | undefined>>;
```

Comment:
i would configure it to aggregate by default, as it seems like an advance use case likely in testing where we'd want per event data, and i'd rather make that harder than making the average use case harder

Reply:
@vladsud made some great points in DMs about the need for developers to have access to per-event data without a lot of configuration. I understand the desire to reduce unnecessary telemetry noise for Fluid/FRS customers, though. How can we meet both needs here? Would it be enough to aggregate by default, but then have webpack-fluid-loader set it to verbose by default? Open to suggestions, but those are the 2 main requirements I want to meet. Another option could be setting it to aggregate by default in AzureClient's configuration of the r11s driver.

Reply:
i wouldn't set if by default anywhere. if someone cares they can set disableTelemetryAggregation: true

Reply:
i don't think it's reasonable to get thousands of log entries per session by default, that is the problem you should fix. i agree supporting getting all that data is good, but it's a niche scenario, so doesn't need easy support